cpp/open3d/visualization/visualizer/O3DVisualizer.cpp

```diff
@@ -829,6 +829,8 @@ struct O3DVisualizer::Impl {
         bool is_default_color;
         bool no_shadows = false;
         Material mat;
+        t::geometry::PointCloud* valid_tpcd = nullptr;
```

Comment:
Is it safe to use a pointer to `t::geometry::PointCloud`?
In the future may we change the pointer-based interface to a reference-based ?